src/lang/en.json

```diff
@@ -0,0 +1,391 @@
+{
+  "blank": "",
+  "nav": {
+    "tab": {
+      "about": "About",
+      "header": "Header",
+      "mime": "MIME types",
+      "security": "Security",
+      "tags": "Tags",
+      "paths": "Paths",
+      "definitions": "Definitions"
+    },
+    "actions": {
+      "delete": "Delete data",
+      "download": {
+        "json": "JSON",
+        "yaml": "YAML"
+      }
+    }
+  },
+  "error": {
+    "title": "Uh oh!",
+    "body": "It seems like something is broken.\nSometimes these issues are caused by an invalid OpenAPI document in the cache. If this is the case, <a href=\"javascript:void(0)\" click.delegate=\"delete()\">deleting the data</a> should fix it.",
```

Comment:
I am not sure that we want to embed so much markup within our localizations. See if you can break this localization into multiple parts or add parameters. Markup should, ideally, live in the template.